samples/CMakeLists.txt

```diff
@@ -1,33 +1,76 @@
 include(BuildHelpers.CMakeLists.txt)
 
-include_directories( 
-    ${CMAKE_SOURCE_DIR}/gameplay/src
-    ${CMAKE_SOURCE_DIR}/external-deps/include
+include_directories(
+        ${CMAKE_SOURCE_DIR}/gameplay/src
```

Comment:
I use CLion for editing and use their 'smart tab' formatting system. Sorry if my liberal use of auto formatting sometimes affects existing formatting that should not be changed.